tests/integration_tests/cmd/test_status.py

```diff
@@ -66,6 +66,7 @@ def test_wait_when_no_datasource(session_cloud: IntegrationCloud, setup_image):
 users:
   - name: something
     ssh-authorized-keys: ["something"]
+  - default
 ca-certs:
```

Comment:
I think we also need to change ca-certs to ca_certs to avoid getting integration test failures in the following test:
```
FAILED tests/integration_tests/cmd/test_status.py::test_status_json_errors - assert 'Deprecated cloud-config provided: users.0.ssh-authorized-keys' in 'Deprecated cloud-config provided: ca-certs:  Deprecated in version 22.3. Use ``ca_certs`` instead., use...
```